test/integration/node-specific/client_encryption.test.ts

```diff
@@ -406,10 +416,10 @@ describe('ClientEncryption integration tests', function () {
     });
 
     it(
-      `does not throw if algorithm has different casing than 'rangePreview'`,
+      `does not throw if algorithm has different casing than 'range'`,
```

Comment:
interesting that this test is "does not throw" rather than "works with any casing" I don't think we have to change something here but I would test for success rather than "not error", what if it ignores this option, does this throw or is it nonsensical usage?